packages/jaeger-ui/package.json

```diff
@@ -26,7 +26,6 @@
     "@types/react-router-dom": "^5.1.0",
     "@types/redux-actions": "2.2.1",
     "@types/object-hash": "^3.0.2",
-    "@types/rollup-plugin-visualizer": "^4.2.1",
```

Comment:
latest version of `rollup-plugin-visualizer` is written in typescript, therefore types are not required anymore